Core/Enums/VideoTags.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace Core.Enums
+{
+	public enum VideoTags
+    {
```

Comment:
Fix tabs, please

Reply:
Tabs are ok. That's how GitHub displays them.